djadmin2/tests/test_core.py

```diff
@@ -9,7 +9,7 @@
 from ..core import Admin2
 
 
-class Thing(models.Model):
+class SmallThing(models.Model):
```

Comment:
AFAR I believe there was clash of names and this was raising some errors for django.